modules/catalog-compaction/src/integrationTest/java/org/apache/ignite/internal/catalog/compaction/ItCatalogCompactionTest.java

```diff
@@ -67,11 +77,71 @@
 class ItCatalogCompactionTest extends ClusterPerClassIntegrationTest {
     private static final int CLUSTER_SIZE = 3;
 
+    /** How often we update the low water mark. */
+    private static final long LW_UPDATE_TIME_MS = TimeUnit.SECONDS.toMillis(10);
```

Comment:
Fixed. Disabled testRaftGroupsUpdate because it became more unstable under already created https://issues.apache.org/jira/browse/IGNITE-23052.

Reply:
@lowka
The idea of the test is to check that 'minActiveTxBeginTime` on all partition listeners updated, not **snapshotted** minActiveTxTime.

And the original `minActiveTxBeginTime` (not `snapshottedMinActiveTxBeginTime`)  was required ONLY for this test.

Since you removed this field (renamed `snapshottedMinActiveTxBeginTime` to `minActiveTxBeginTime` ) and since now we have full integration test for catalog compaction runner, I suggest to remove `testRaftGroupsUpdate` because it will not work properly until fully reworked, and, again we already have full it-test for this .

Reply:
@lowka
The idea of the test testRaftGroupsUpdate is to check that 'minActiveTxBeginTime` on all partition listeners updated, not **snapshotted** minActiveTxTime.

And the original minActiveTxBeginTime (not snapshottedMinActiveTxBeginTime) was required ONLY for this test.

Since we removed this field (renamed snapshottedMinActiveTxBeginTime to minActiveTxBeginTime ) and since now we have full integration test for catalog compaction runner, I suggest to remove testRaftGroupsUpdate because it will not work properly until reworked, and, again seems we already have test for this .

Reply:
Done.